console/src/components/Services/Data/TableModify/ColumnEditorList.js

```diff
@@ -104,6 +105,31 @@ const ColumnEditorList = ({
 
       const isOk = getConfirmation(confirmMessage, true, colName);
       if (isOk) {
+        const pk = [];
+        let flag = true;
+        for (const [_, value1] of tableSchema.primary_key.columns.entries()) {
+          for (const [index2, value2] of tableSchema.columns.entries()) {
+            if (value2.column_name.localeCompare(colName) == 0) {
+              flag = false;
+            } else {
+              if (
+                flag == true &&
+                value1.localeCompare(value2.column_name) == 0
+              ) {
+                pk.push(index2);
+              } else if (value1.localeCompare(value2.column_name) == 0) {
+                pk.push(index2 - 1);
+              }
+            }
+          }
+          flag = true;
+        }
+        const pks = [];
+        for (const [_, value1] of pk.sort().entries()) {
+          pks.push(value1.toString());
+        }
```

Comment:
You could do a similar operation like so : 
```suggestion
       const pks = [...pk.sort()];
```

Reply:
Also, I wanted to understand why `.sort()` is required?